src/cogent3/core/new_alignment.py

```diff
@@ -3471,7 +3399,8 @@ def _(self, key: str) -> Union[new_sequence.Sequence, Aligned]:
 
     def __repr__(self) -> str:
         one_seq = self[self.parent.names[0]]
-        return f"({one_seq!r}, {self.parent.num_seqs - 1} x seqs)"
+        one_seq = f"{one_seq!r}, ... " if self.parent.num_seqs > 1 else f"{one_seq!r}"
+        return f"[{one_seq}], {self.parent.num_seqs} x seqs"
```

Comment:
use enclosing `()` not `[]` surrounding the entire string. I wanted to communicate that this is not a mutable object.